org.eclipse.mylyn-target/mylyn-e4.26.target

```diff
@@ -71,6 +71,22 @@
 		</location>
 		<location includeDependencyDepth="infinite" includeDependencyScopes="compile,provided,runtime" includeSource="true" missingManifest="generate" type="Maven">
 			<dependencies>
+				<dependency>
+				    <groupId>com.sun.xml.bind</groupId>
+				    <artifactId>jaxb-impl</artifactId>
+				    <version>2.3.3</version>
+				</dependency>
+				<dependency>
+				    <groupId>jakarta.xml.bind</groupId>
```

Comment:
we have this in the target already from `https://download.eclipse.org/tools/orbit/downloads/drops/R20221123021534`